uat/testing-features/src/main/java/com.aws.greengrass/CloudWatchSteps.java

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
Didn`t change this file, just move location

Reply:
This comment is applicable for other files under this folder.

The file path has to be `uat/testing-features/src/main/java/com/aws/greengrass/CloudWatchSteps.java` instead of `uat/testing-features/src/main/java/com.aws.greengrass/CloudWatchSteps.java`

Reply:
This change did not remove the previous steps/resources files. It added these new ones only. Is that what you wanted to do too?

Reply:
Yeah I believe we should remove them from the old place